dxw/advisories
# Feature/enhance scrolling experience

This pull request (PR) fixes the problem of the screen flickering when scrolling on pages that don’t have enough content to fill up the whole height of the viewport.

The height of the page would drastically reduce due to the applied `.sticky` class changing the height of the nav bar, causing the class to be constantly removed and added on scroll. To resolve this issue, the body has been set to occupy the full screen height and the main landmark to grow and fill any empty space. 
 
## Demo
**Before:**

https://user-images.githubusercontent.com/53922624/227529121-309de2fb-807d-49ce-a7dc-80a38ff1a3fd.mp4


**After:**


https://user-images.githubusercontent.com/53922624/227574559-9ddb2953-e9c1-4fee-9731-6f10fe76b3f8.mp4


## Testing

1. Go to [dxw advisories – WordPress security advisories, audit and assurance](https://advisories.dxw.com/)
1. Put something ridiculous in the search bar, like fibble
1. Press the search button, this should take you to an empty search results page
1. Scroll down hard, to the bottom of the page
1. Make sure that the flickering behaviour has stopped
